egs/wsj/s5/steps/libs/nnet3/report/log_parse.py

```diff
@@ -391,6 +391,70 @@ def parse_prob_logs(exp_dir, key='accuracy', output="output"):
     return map(lambda x: (int(x), float(train_objf[x]),
                           float(valid_objf[x])), iters)
 
+def parse_rnnlm_prob_logs(exp_dir, key='objf'):
+    train_prob_files = "%s/log/train.*.*.log" % (exp_dir)
+    valid_prob_files = "%s/log/compute_prob.*.log" % (exp_dir)
+    train_prob_strings = common_lib.get_command_stdout(
+        'grep -e {0} {1}'.format(key, train_prob_files))
+    valid_prob_strings = common_lib.get_command_stdout(
+        'grep -e {0} {1}'.format(key, valid_prob_files))
+
+    # LOG
```

Comment:
what is this comment for?

Reply:
They are example strings to be matched for the regular expression.